src/ontology/mondo-edit.obo

```diff
@@ -204870,6 +204870,7 @@ id: MONDO:0010574
 name: syndromic X-linked intellectual disability 5
 def: "X-linked Dandy-Walker malformation with intellectual disability, basal ganglia disease and seizures (XDIBS), or Pettigrew syndrome is a central nervous system malformation characterized by severe intellectual deficit, early hypotonia with progression to spasticity and contractures, choreoathetosis, seizures, dysmorphic face (long face with prominent forehead), and brain imaging abnormalities such as Dandy-Walker malformation, and iron deposition." [Orphanet:1568]
 comment: Editor note: check relationship to friend syndrome
+subset: ordo_malformation_syndrome {source="Orphanet:85329"}
```

Comment:
can you merge these together? ie add source="Orphanet:85329" on the other subset annotation?